doc/user/separate.xml

```diff
@@ -55,12 +55,12 @@
   and header files differ.  If you build "Controller A" first,
   then "Controller B", on the second build everything would
   have to be rebuilt, because &SCons; sees that the build
-  instructions differ, and thus the targets that depend on those
-  different instructions are not valid for the current build.
+  instructions are different to the ones used to build the
```

Comment:
"different from those used in the previous build of those targets, and so the targets need to be rebuilt" ?

Reply:
updated